src/System.Runtime.InteropServices.RuntimeInformation/src/System/Runtime/InteropServices/RuntimeInformation/RuntimeInformation.cs

```diff
@@ -9,12 +9,7 @@ namespace System.Runtime.InteropServices
 {
     public static partial class RuntimeInformation
     {
-#if uapaot
-        private const string FrameworkName = ".NET Native";
-#else // uap || netcoreapp
```

Comment:
What about places that check for this, e.g. AssertExtensions?

Reply:
Can you elaborate?

Reply:
e.g. AssertExtensions.Throws checks to see whether the current platform is .NET Native.  It now never will be.

Reply:
Thanks I missed that pattern. Will remove that as well.